samtranslator/swagger/swagger.py

```diff
@@ -788,6 +789,32 @@ def add_models(self, models):
 
             self.definitions[model_name.lower()] = schema
 
+    def add_resource_policy(self, resource_policy):
+        """
+        Add resource policy definition to Swagger.
+
+        :param dict resource_policy: Dictionary of resource_policy statements which gets translated
+        :return:
+        """
+        if resource_policy is None:
+            return
+
+        custom_statements = resource_policy.get('CustomStatements')
+        if custom_statements is not None:
+            self.resource_policy['Version'] = '2012-10-17'
+            if self.resource_policy.get('Statement') is None:
+                self.resource_policy['Statement'] = custom_statements
+            else:
+                statement = self.resource_policy['Statement']
+                if type(statement) is list:
+                    statement.extend(custom_statements)
+                else:
+                    statement = [statement, custom_statements]
```

Comment:
If `custom_statements` is a list, doesn't this mean it'll be a nested list inside the outer list? Is that what we want?